.evergreen/run-orchestration.sh

```diff
@@ -98,10 +114,11 @@ if ! curl --silent --show-error --data @"$ORCHESTRATION_FILE" "$ORCHESTRATION_UR
   exit 1
 fi
 cat tmp.json
-URI=$(python -c 'import json; j=json.load(open("tmp.json")); print(j["mongodb_auth_uri" if "mongodb_auth_uri" in j else "mongodb_uri"])' | tr -d '\r')
+
+URI=$(${PYTHON:-python} -c 'import json; j=json.load(open("tmp.json")); print(j["mongodb_auth_uri" if "mongodb_auth_uri" in j else "mongodb_uri"])' | tr -d '\r')
```

Comment:
Can `PYTHON=python` be set above near/in the `if [ -n "$BASH" ]; then` block above? e.g.
```bash
export PYTHON
if [[ -n "$BASH" ]]; then
  . $DIR/find-python3.sh
  echo "Finding Python3 binary..."
  PYTHON="$(find_python3 2>/dev/null)"
  echo "Finding Python3 binary... done."
else
  # A note about Docker-provided Python if not running in Bash?
  PYTHON=python
fi

# ...

URI=$(${PYTHON:?} -c '...')
```

Reply:
Updated, with notes